examples/step-13/step-13.cc

```diff
@@ -510,30 +506,30 @@ namespace Step13
     class Base
     {
     public:
-      Base (Triangulation<dim> &coarse_grid);
-      virtual ~Base ();
+      Base(Triangulation<dim> &coarse_grid);
+      virtual ~Base();
 
-      virtual void solve_problem () = 0;
-      virtual void postprocess (const Evaluation::EvaluationBase<dim> &postprocessor) const = 0;
-      virtual void refine_grid () = 0;
-      virtual unsigned int n_dofs () const = 0;
+      virtual void solve_problem() = 0;
+      virtual void postprocess(
+        const Evaluation::EvaluationBase<dim> &postprocessor) const = 0;
+      virtual void         refine_grid()                            = 0;
+      virtual unsigned int n_dofs() const                           = 0;
```

Comment:
Interesting. I don't think I had seen this before. I guess I can see the logic here -- it aligns the function names and the `=0`. Interestingly, it only does it for the last two functions. That's not consistent, but I think it's ok.